services/audit-service/src/main/java/datawave/microservice/audit/AuditController.java

```diff
@@ -122,17 +122,22 @@ private boolean sendMessage(AuditParameters parameters) {
             }
             
             boolean success = messageChannel.send(MessageBuilder.withPayload(AuditMessage.fromParams(parameters)).setCorrelationId(auditId).build());
-            
+
+            // Prevent a 2xx response code to the caller.
+            if (!success) {
```

Comment:
When this ticket was created the audit service had no concept of writing audit messages that failed to publish to the message channel to disk.  We have come a long way since then, so now when an audit message fails to send, we attempt to write it to disk before returning a failed status.  If we start throwing an exception here, we completely skip the logic that attempts to write the audit message to disk.  

The reality is that this was a stale ticket that we resolved a while ago which should have been closed.

Reply:
Understood, I'll modify the commit to only address cleaning up the boolean logic that you noted below, and revert my other changes.

Reply:
Awesome!  Thanks!